doc/glossary.rst

```diff
@@ -43,12 +45,22 @@ general neuroimaging concepts. If you think a term is missing, please consider
         a type, such as gradiometer, and a unit, such as Tesla/Meter that
         is used in the code base, e.g. for plotting.
 
-    Digitization
-        Digitization is a procedure of recording the headshape of a subject and the fiducial coils and/or eeg electrodes locations on the subject’s head. They are represented as a set of points in a 3D space. See :ref:`reading-dig-montages` and :ref:`dig-formats`.  
+    digitization
+        Digitization is a procedure of recording the headshape of a subject and
+        the fiducial coils (or :term:`HPI`) and/or eeg electrodes locations on
+        the subject’s head. They are represented as a set of points in a 3D space.
+        See :ref:`reading-dig-montages` and :ref:`dig-formats`.  
 
     dipole
         See :term:`equivalent current dipole`.
 
+    eLORETA and sLORETA
+        eLORETA and sLORETA (exact and standardized low resolution brain
+        electromagnetic tomography) are linear source estimation techniques,
+        as are dSPM or MNE (:term:`minimum-norm estimation`). sLORETA outputs
+        standardized values such as dSPM, while eLORETA outputs current density
+        estimates like MNE. See :func:`mne.minimum_norm.apply_inverse`.
```

Comment:
Our implementation of `eLORETA` outputs some normalized quantity (unfortunately), not current density estimates.

Reply:
See https://github.com/mne-tools/mne-python/pull/6050

Reply:
Should the definition be updated clarifying that?

Reply:
I would not say it outputs current density here. Maybe just call them "normalized estimates"? And if we say it's current density in our minimum-norm/eLORETA docs somewhere you can change that, too

Reply:
just remove this sentence then: `, while eLORETA outputs current density  estimates like MNE`